ui-v2/src/components/concurrency/task-run-concurrenct-view/dialogs/create-dialog.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { CreateLimitDialog } from "./create-dialog";
+
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { createWrapper } from "@tests/utils";
+import { expect, test, vi } from "vitest";
+
+const MOCK_DATA = {
+	id: "0",
+	created: "2021-01-01T00:00:00Z",
+	updated: "2021-01-01T00:00:00Z",
+	tag: "my tag 0",
+	concurrency_limit: 1,
+	active_slots: [] as Array<string>,
+};
+
+test.skip("CreateLimitDialog calls onSubmit upon entering form data", async () => {
```

Comment:
Skipping this test for now.
For some reason it passes on my machine, but fails in CI

<img width="167" alt="Screenshot 2024-12-11 at 12 00 54 PM" src="https://github.com/user-attachments/assets/6524888e-8c89-4737-b04f-bfd9f1409558" />
![Screenshot 2024-12-11 at 11 59 03 AM](https://github.com/user-attachments/assets/c40b9ca6-305b-4e50-997b-8f2682289dbc)

Reply:
Could it be because the `ResizeObserverMock` isn't in a `beforeAll`?

Reply:
that didn't fix it.
I'm pretty stumped on this one

Reply:
Yeah, that's really weird. Especially considering the tests for the other dialogs are fine 🫠